res/database/Default/database_ads.xml

```diff
@@ -252,8 +253,8 @@ You will really have to chew.</en>
 				<de>Big Kahuna Burger</de>
 			</title>
 			<description>
-				<de>Mmmmhmmm - das IST ein leckerer Burger. Wenn Du auf Burger stehst, mußt Du die probieren!</de>
-				<en>Mmmmhmmm - this is a delicious burger. If you are on burgers, you have to try them!</en>
+				<de>Mmmmhmmm - das IST ein leckerer Burger. Wenn Du auf Burger stehst, musst Du die probieren!</de>
+				<en>Mmmmhmmm - this IS a delicious burger. If you're into burgers, you have to try these!</en>
```

Comment:
Hmm... formal English = "are" instead of "'re". Think we need to decide for a "writing style". Exception here might be the "slang"-speek or ... "cited"-style.

Reply:
fine by me either way, just "on burgers" is wrong.